src/domain/ledger/deposit-fee-calculator.ts

```diff
@@ -0,0 +1,12 @@
+import { toSats } from "@domain/bitcoin"
+
+export const DepositFeeCalculator = (amount: Satoshis): DepositFeeCalculator => {
```

Comment:
Hmm... where would you put it? I was thinking because the `ledger` is where the `fee` gets recorded. But I could also see a case for putting it under `wallet`. What is your preference?

Reply:
I think is wallet responsibility